src/drivers/cloudflare-kv-http.ts

```diff
@@ -96,13 +96,21 @@ export default defineDriver<KVHTTPOptions>((opts) => {
   const kvFetch = $fetch.create({ baseURL, headers })
 
   const hasItem = async (key: string) => {
-    const { success } = await kvFetch(`/values/${key}`)
-    return success
+    try {
+      await kvFetch(`/values/${key}`)
```

Comment:
(not directly related to this PR) We might use meta-data endpoint for existance instead of getting value.